openapi/sled-agent.json

```diff
@@ -1505,6 +1505,20 @@
               "type"
             ]
           },
+          {
+            "type": "object",
+            "properties": {
+              "type": {
+                "type": "string",
+                "enum": [
+                  "switch_zone_setup"
+                ]
+              }
+            },
+            "required": [
+              "type"
+            ]
+          },
```

Comment:
(I had to add this by hand because sled-agent depends on sled-agent-client. We should break this cycle -- the server shouldn't depend on the client, and instead we should add a sled-agent-common that has the common types shared by the client and server.)

Reply:
Will do this in a followup.